sixtyfps_compiler/literals.rs

```diff
@@ -64,8 +64,8 @@ fn test_parse_color_literal() {
 }
 
 pub fn unescape_string(string: &str) -> Option<String> {
-    if string.contains('\n') {
-        // FIXME: new line in string literal not yet supported
+    if string.contains('\n') || string.contains('\t') {
```

Comment:
I think I shouldn't be adding this, tab characters themselves should be possible to add in string literals.

But since I also don't understand why \n wouldn't be supported, I wanted to ask first.

Reply:
the reason why we don't support line return in string literal is that we haven't made our mind of what the semantic would be. is a \n considered as a space or a \n, what about the identation.
I guess \t can stay \t, so i guess this change should indeed not be added.

Reply:
Removed